### ISSUE TITLE
Bug: Blockcontent references in clean install (template) produce "reference search failed" - expected } following object body

### ISSUE THREAD

**Issue by CubertFarnsworth** [sanity-io/sanity#2698]
**Describe the bug**

Error occurs in clean template with no edits. 
Seems to apply to "BlockContent" types.

**To Reproduce**

1. Init with movie template
2. Click on 'Screening' under "Content"
3. Select a screening.
3. Try selecting "Movie" dropdown

**Expected behavior**

Expected references to work. 

**Screenshots**
![bug](https://user-images.githubusercontent.com/12510574/130698836-3a556a15-1793-437b-80f0-879be8dd04ec.png)

**Which versions of Sanity are you using?**

Run `sanity versions` in the terminal and copy-paste the result here.
```
@sanity/cli                2.16.0 (up to date)
@sanity/base               2.16.0 (up to date)
@sanity/components         2.14.0 (up to date)
@sanity/core               2.16.0 (up to date)
@sanity/default-layout     2.16.0 (up to date)
@sanity/default-login      2.16.0 (up to date)
@sanity/desk-tool          2.16.0 (up to date)
@sanity/google-maps-input  2.16.0 (up to date)
@sanity/vision             2.15.0 (up to date)
```

**What operating system are you using?**
`OS Name:                   Microsoft Windows 10 Enterprise`
`OS Version:                10.0.19043 N/A Build 19043`

**Which versions of Node.js / npm are you running?**

Run `npm -v && node -v` in the terminal and copy-paste the result here.
`NPM: 6.14.14`
`Node: v14.17.5`

**Additional context**

**Security issue?**
`No`

**Comment by CubertFarnsworth** [comment]
When clicking the "link" icon the left side of the input-box this error is found in the console window:
(Still on clean template with no changes or edits from me)

![bug2](https://user-images.githubusercontent.com/12510574/130699041-b8f92704-bb6a-4f5b-bd65-5391de980d2d.png)

**Comment by geoffreyjball** [comment]
Thanks for reporting, @CubertFarnsworth, and sorry for the inconvenience. This issue is resolved by release 2.16.1.
